test/dates.t.cpp

```diff
@@ -52,7 +52,7 @@ void testInit (UnitTest& t, const std::string& value, Datetime& var)
 ////////////////////////////////////////////////////////////////////////////////
 int main (int, char**)
 {
-  UnitTest t (164);
```

Comment:
Hang on, this just changes the number of expected tests. Let's add a test case for this?

Reply:
The currently failing testcase is [here](https://github.com/GothenburgBitFactory/libshared/blob/ee050a2cc68c46e8748113246b26feb0ad9f34c5/test/dates.t.cpp#L239):
```
t.ok (eopww < soww, "eopww < soww");
```
in `dates.t.cpp`. It only fails on a Saturday though...

Reply:
The change her only aligns the number of actually present testcases with the number of planned ones (i.e. the argument of the `UnitTest` constructor).

Reply:
Having a specific test case for "`eopww` should be before `soww` on Saturdays" would require the reference date (a.k.a. `now`) to be injectable into the Datetime parser, which would be preferable, but a bigger endeveaour... 🤔

Reply:
Ah, I see -- weird to have date-time tests depend on the _actual_ date and time. Yeah, fuzzing that (or just replacing them with fake time and figuring out the right times to test all the corner cases) sounds like a good, but larger project.